Makefile

```diff
@@ -26,15 +27,20 @@ endif
 
 obj-$(CONFIG_LUNATIK) += lunatik.o
 
-lunatik-objs += lua/lapi.o lua/lcode.o lua/lctype.o lua/ldebug.o lua/ldo.o \
-	 lua/ldump.o lua/lfunc.o lua/lgc.o lua/llex.o lua/lmem.o \
-	 lua/lobject.o lua/lopcodes.o lua/lparser.o lua/lstate.o \
-	 lua/lstring.o lua/ltable.o lua/ltm.o \
-	 lua/lundump.o lua/lvm.o lua/lzio.o lua/lauxlib.o lua/lbaselib.o \
-	 lua/lbitlib.o lua/lcorolib.o lua/ldblib.o lua/lstrlib.o \
-	 lua/ltablib.o lua/lutf8lib.o lua/loslib.o lua/lmathlib.o lua/linit.o \
-	 lua/loadlib.o \
-	 arch/$(ARCH)/setjmp.o util/modti3.o lunatik_core.o
+lua-objs = lua/lapi.o lua/lcode.o lua/lctype.o lua/ldebug.o lua/ldo.o \
+	lua/ldump.o lua/lfunc.o lua/lgc.o lua/llex.o lua/lmem.o \
+	lua/lobject.o lua/lopcodes.o lua/lparser.o lua/lstate.o \
+	lua/lstring.o lua/ltable.o lua/ltm.o \
+	lua/lundump.o lua/lvm.o lua/lzio.o lua/lauxlib.o lua/lbaselib.o \
+	lua/lbitlib.o lua/lcorolib.o lua/ldblib.o lua/lstrlib.o \
+	lua/ltablib.o lua/lutf8lib.o lua/loslib.o lua/lmathlib.o lua/linit.o \
+	lua/loadlib.o luautil.o
```

Comment:
why changing all these lines?

Reply:
To put all lua related objects in on variable on the Lunatik building

Reply:
you need change the indentation of this lines to do that.. please, take cara of that..